src/bsp/src/stm32/hal/src/stm32h735zg/hal_init.c

```diff
@@ -82,8 +82,8 @@ static void Hal_initCache() {
     /* Enable I-Cache */
     SCB_EnableICache();
 
-    /* Enable D-Cache */
-    SCB_EnableDCache();
+    /* Do not enable D-Cache as it causes a hardfault when trying to read or write to flash */
+    // SCB_DisableDCache();
```

Comment:
Is the disable supposed to be commented? Or it's disabled by default?

Reply:
I just reenabled the cache